chapters/substitution.adoc

```diff
@@ -2,13 +2,25 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. Robot substitution during game play:
+.. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+.. A robot can only be substituted while obeying the following specifications:
+... Is at least on top of any touch line, if possible the robot can also be placed beyond the field boundaries
+... At a distance from the halfway line that must not exceed 30 centimeters
+.. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs the following operations.
+... Put robots in. If the ball is at least 2 meters away from the halfway line, the robot can be substituted without notifying the <<Referee, referee>>.
+... Take robots out. If the robot is placed at the previous specified position and the ball is at least 2 meters away from the halfway line, the robot can be removed without notifying the <<Referee, referee>>.
+
+. Robot substitution on next stoppage:
+.. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs the following operations.
+... Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
+... Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
```

Comment:
With the new rules, "the next opportunity" does not make sense anymore, because you can now add robots whenever you want.

The rule was added to motivate the automatic interchange. With new changes, there is an even higher motivation for this, because you can substitute robots immediately, so I think it is fine to drop this line.

Reply:
Makes sense, but what if some team still wants to manually replace a robot? For example, a robot that for some reason got a low battery and cannot move, it won't cause any damage to itself or other robots, thus, a emergency stop is not required, but it will still need to be manually removed from the field, this kind of situation can wait until a game stoppage

Reply:
The 1 meter rule doesn't make much sense, since now it is possible to put robots in at any time, so I'll just remove it

Reply:
Without #16, teams can still substitute robots during stop as usual. No need for an emergency stop.